Header header
float32 rssi_modem
float32 integrity_modem
float32 rssi_usbl
float32 integrity_usbl
float32 distance

